pkgs/by-name/li/libsrm/package.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  unstableGitUpdater,
+}:
+
+stdenv.mkDerivation {
+  name = "libsrm";
+  version = "0-unstable-2020-11-05";
+
+  src = fetchFromGitHub {
+    owner = "ka-petrov";
+    repo = "LibSRM";
+    rev = "7db2df68147ea07c93be0973fb04da7e4bbdaa11";
+    hash = "sha256-FYKlu9MqXjVr6eiHYOPsypypXmQtC8mlfCf0G9Lbt94=";
+  };
+
+  postPatch = ''
+    echo "install(TARGETS srm DESTINATION lib)" >> libsrm/CMakeLists.txt
+    echo "install(TARGETS srm_test DESTINATION bin)" >> libsrm/CMakeLists.txt
+    cd libsrm
```

```suggestion
    echo "install(TARGETS srm DESTINATION lib)" >> CMakeLists.txt
    echo "install(TARGETS srm_test DESTINATION bin)" >> CMakeLists.txt
```
I think you can set `sourceRoot = "libsrm";` to do this automatically. Package LGTM otherwise.

---

`sourceRoot = "libsrm"` does not work, but I have modified the derivation to use the correct `sourceRoot`.